codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - "root/project::"
```

Comment:
nit: add new line.

For my education; why was this needed?

Reply:
When reported from CircleCI the paths are prefixed with `root/project` making it so that they couldn't be found relative to the root of the repository. This replaces that per https://docs.codecov.io/v4.3.6/docs/fixing-paths